src/dynamics/video_player/player/player.js

```diff
@@ -661,7 +661,7 @@ Scoped.define("module:VideoPlayer.Dynamics.Player", [
                     ) {
                         if (fullscreened) return false;
                         this.set("floatingsidebar", !hideSidebar && isFloating && withSidebar);
-                        this.set("gallerysidebar", !hideSidebar && !isFloating && (Types.is_defined(mobileViewport) && !mobileViewport));
+                        this.set("gallerysidebar", !hideSidebar && !isFloating && (Types.is_defined(mobileViewport) && !mobileViewport) && !this.get('outstream'));
```

Comment:
Hi @Josewd, instead of using `this.get('outstream')` I suggest listing it as a dependency on the computed variable:
`"show_sidebar:hide_sidebar,is_floating,with_sidebar,fullscreened,mobileviewport,outstream"`
This way the function will automatically run again if `outstream` is changed.

Reply:
Done, thanks @ntshcalleia